operations/query.rb

```diff
@@ -0,0 +1,5 @@
+module Query
+  def self.build_query(params)
+    params.map{|key, value| "#{key}=#{value}"}.join('&')
```

Comment:
`params.map { |key, value| "#{key}=#{value}" }.join('&')`
Little things like spaces matter a lot, makes it easier to read & reason about the code

Reply:
Also, consider using the following pattern for query / service / operation objects

```ruby
def ApplicationService
  def self.call(*args)
    new(*args).call
  end
end

class QueryBuilder < ApplicationService
  def initialize(params)
    ...
  end

  def call
    ..
  end
end

QueryBuilder.call(params) #=> ...
```

Also, take a look & investigate a little - https://github.com/pivorakmeetup/pivorak-web-app/tree/development/app/services